lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQuerySqlProvider.java

```diff
@@ -363,6 +497,30 @@ private static String commonTaskSecondaryObjectReferencesWhereStatement() {
         + "</if>";
   }
 
+  private static String commonTaskSecondaryObjectReferencesWhereStatementOracle() {
+    return "<if test='secondaryObjectReferences != null'>"
+        + "AND (<foreach item='item' collection='secondaryObjectReferences' separator=' OR '> "
+        + "<if test='item.company != null'>o2.COMPANY = #{item.company} </if>"
+        + "<if test='item.system != null'> "
+        + "<if test='item.company != null'>AND</if> "
+        + "o2.SYSTEM = #{item.system} </if>"
+        + "<if test='item.systemInstance != null'> "
+        + "<if test='item.company != null or item.system != null'>AND</if> "
+        + "o2.SYSTEM_INSTANCE = #{item.systemInstance} </if>"
+        + "<if test='item.type != null'> "
+        + "<if test='item.company != null or item.system != null or item.systemInstance != null'>"
+        + "AND</if> "
+        + "o2.TYPE = #{item.type} </if>"
+        + "<if test='item.value != null'> "
+        + "<if test='item.company != null or item.system != null "
+        + "or item.systemInstance != null or item.type != null'>"
+        + "AND</if> "
+        + "o2.VALUE = #{item.value} "
+        + "</if>"
+        + "</foreach>)"
+        + "</if>";
+  }
```

Comment:
this is a direct consequence of `commonTaskWhereStatementOracle`, right?

Reply:
yes its removed